{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Failed to start server: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:369:69)\n    at Parser.handlePacket (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:187:21)\n    at Parser.parse (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:102:30)\n    at Socket.<anonymous> (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\index.ts:7:48)\n    at Socket.emit (node:events:524:28)\n    at addChunk (node:internal/streams/readable:561:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)\n    at Socket.Readable.push (node:internal/streams/readable:392:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)","timestamp":"2025-06-11T02:23:10.583Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Failed to start server: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\dist\\parser.js:285:98)\n    at Parser.handlePacket (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\dist\\parser.js:122:29)\n    at Parser.parse (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\dist\\parser.js:35:38)\n    at Socket.<anonymous> (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\dist\\index.js:11:42)\n    at Socket.emit (node:events:524:28)\n    at addChunk (node:internal/streams/readable:561:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)\n    at Readable.push (node:internal/streams/readable:392:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)","timestamp":"2025-06-11T02:23:46.694Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Failed to start server: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:369:69)\n    at Parser.handlePacket (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:187:21)\n    at Parser.parse (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:102:30)\n    at Socket.<anonymous> (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\index.ts:7:48)\n    at Socket.emit (node:events:524:28)\n    at addChunk (node:internal/streams/readable:561:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)\n    at Socket.Readable.push (node:internal/streams/readable:392:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)","timestamp":"2025-06-11T02:24:38.950Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Failed to start server: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:369:69)\n    at Parser.handlePacket (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:187:21)\n    at Parser.parse (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:102:30)\n    at Socket.<anonymous> (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\index.ts:7:48)\n    at Socket.emit (node:events:524:28)\n    at addChunk (node:internal/streams/readable:561:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)\n    at Socket.Readable.push (node:internal/streams/readable:392:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)","timestamp":"2025-06-11T02:26:59.203Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Failed to start server: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:369:69)\n    at Parser.handlePacket (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:187:21)\n    at Parser.parse (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:102:30)\n    at Socket.<anonymous> (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\index.ts:7:48)\n    at Socket.emit (node:events:524:28)\n    at addChunk (node:internal/streams/readable:561:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)\n    at Socket.Readable.push (node:internal/streams/readable:392:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)","timestamp":"2025-06-11T02:30:08.592Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Error connecting to main database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:369:69)\n    at Parser.handlePacket (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:187:21)\n    at Parser.parse (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:102:30)\n    at Socket.<anonymous> (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\index.ts:7:48)\n    at Socket.emit (node:events:524:28)\n    at addChunk (node:internal/streams/readable:561:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)\n    at Socket.Readable.push (node:internal/streams/readable:392:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)","timestamp":"2025-06-17T16:30:38.246Z"}
{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"324","message":"Error connecting to main database: password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:369:69)\n    at Parser.handlePacket (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:187:21)\n    at Parser.parse (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\parser.ts:102:30)\n    at Socket.<anonymous> (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\New folder\\node_modules\\pg-protocol\\src\\index.ts:7:48)\n    at Socket.emit (node:events:524:28)\n    at addChunk (node:internal/streams/readable:561:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)\n    at Socket.Readable.push (node:internal/streams/readable:392:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)","timestamp":"2025-06-17T16:42:59.945Z"}
{"0":{"context":{"key":"password","label":"password","regex":{},"value":"securepassword123"},"message":"\"password\" with value \"securepassword123\" fails to match the required pattern: /^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@$!%*?&])[A-Za-z\\d@$!%*?&]{8,}$/","path":["password"],"type":"string.pattern.base"},"level":"error","message":"Validation error:","timestamp":"2025-06-19T20:14:25.945Z"}
{"0":{"context":{"key":"password","label":"password","regex":{},"value":"se@curepassword123"},"message":"\"password\" with value \"se@curepassword123\" fails to match the required pattern: /^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@$!%*?&])[A-Za-z\\d@$!%*?&]{8,}$/","path":["password"],"type":"string.pattern.base"},"level":"error","message":"Validation error:","timestamp":"2025-06-19T20:14:43.858Z"}
{"0":{"context":{"key":"password","label":"password","regex":{},"value":"securepassword123"},"message":"\"password\" with value \"securepassword123\" fails to match the required pattern: /^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@$!%*?&])[A-Za-z\\d@$!%*?&]{8,}$/","path":["password"],"type":"string.pattern.base"},"level":"error","message":"Validation error:","timestamp":"2025-06-19T20:14:54.163Z"}
{"0":{"context":{"key":"password","label":"password","regex":{},"value":"securepassword1234"},"message":"\"password\" with value \"securepassword1234\" fails to match the required pattern: /^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@$!%*?&])[A-Za-z\\d@$!%*?&]{8,}$/","path":["password"],"type":"string.pattern.base"},"level":"error","message":"Validation error:","timestamp":"2025-06-19T20:16:04.592Z"}
{"0":{"context":{"key":"password","label":"password","regex":{},"value":"securepassword123!"},"message":"\"password\" with value \"securepassword123!\" fails to match the required pattern: /^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@$!%*?&])[A-Za-z\\d@$!%*?&]{8,}$/","path":["password"],"type":"string.pattern.base"},"level":"error","message":"Validation error:","timestamp":"2025-06-19T20:16:46.597Z"}
{"level":"error","message":"Registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresDriver.js:875:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:69:18)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:57:31)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\middleware\\validation.middleware.js:15:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2025-06-19T20:17:09.781Z"}
{"level":"error","message":"Registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresDriver.js:875:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:69:18)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:57:31)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\middleware\\validation.middleware.js:15:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2025-06-19T20:20:51.835Z"}
{"level":"error","message":"Registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresDriver.js:875:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:69:18)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:57:31)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\middleware\\validation.middleware.js:15:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2025-06-19T20:25:31.550Z"}
{"level":"error","message":"Registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresDriver.js:875:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:69:18)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:57:31)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\middleware\\validation.middleware.js:15:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2025-06-19T20:27:39.534Z"}
{"level":"error","message":"Registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresDriver.js:875:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:69:18)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:57:31)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\middleware\\validation.middleware.js:15:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2025-06-19T20:35:49.835Z"}
{"level":"error","message":"Registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresDriver.js:875:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:69:18)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:57:31)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\middleware\\validation.middleware.js:15:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2025-06-19T20:44:07.103Z"}
{"level":"error","message":"Registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresDriver.js:875:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:69:18)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:57:31)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\middleware\\validation.middleware.js:15:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2025-06-19T20:48:11.923Z"}
{"level":"error","message":"Registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresDriver.js:875:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:69:18)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:57:31)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\middleware\\validation.middleware.js:15:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2025-06-19T20:49:23.781Z"}
{"level":"error","message":"Registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresDriver.js:875:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:69:18)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:57:31)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\middleware\\validation.middleware.js:15:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2025-06-19T21:02:20.939Z"}
{"level":"error","message":"Registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresDriver.js:875:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:69:18)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:57:31)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\middleware\\validation.middleware.js:15:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2025-06-19T21:04:49.045Z"}
{"level":"error","message":"Registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresDriver.js:875:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:69:18)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:57:31)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\middleware\\validation.middleware.js:15:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2025-06-19T21:06:57.276Z"}
{"level":"error","message":"Registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresDriver.js:875:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:69:18)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:57:31)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\middleware\\validation.middleware.js:15:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2025-06-19T21:12:28.480Z"}
{"level":"error","message":"Registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresDriver.js:875:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:69:18)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:57:31)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\middleware\\validation.middleware.js:15:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2025-06-19T21:13:57.084Z"}
{"level":"error","message":"Registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresDriver.js:875:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:69:18)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:57:31)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\middleware\\validation.middleware.js:15:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2025-06-19T21:15:38.609Z"}
{"level":"error","message":"Registration error: No metadata for \"Company\" was found.","stack":"EntityMetadataNotFoundError: No metadata for \"Company\" was found.\n    at DataSource.getMetadata (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\data-source\\DataSource.js:305:19)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\persistence\\EntityPersistExecutor.js:69:26\n    at Array.forEach (<anonymous>)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\persistence\\EntityPersistExecutor.js:62:26\n    at Array.map (<anonymous>)\n    at EntityPersistExecutor.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\persistence\\EntityPersistExecutor.js:59:66)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:63:17)","timestamp":"2025-06-19T21:21:03.328Z"}
{"code":"23502","column":"domain","detail":"Failing row contains (45582670-6e9f-415c-99c1-1267b2a10512, Acme Corp, null, t, null, 2025-06-20 02:53:32.507368, 2025-06-20 02:53:32.507368).","driverError":{"code":"23502","column":"domain","detail":"Failing row contains (45582670-6e9f-415c-99c1-1267b2a10512, Acme Corp, null, t, null, 2025-06-20 02:53:32.507368, 2025-06-20 02:53:32.507368).","file":"execMain.c","length":316,"line":"1988","name":"error","routine":"ExecConstraints","schema":"public","severity":"ERROR","table":"companies"},"file":"execMain.c","length":316,"level":"error","line":"1988","message":"Registration error: null value in column \"domain\" of relation \"companies\" violates not-null constraint","parameters":["Acme Corp"],"query":"INSERT INTO \"companies\"(\"id\", \"name\", \"domain\", \"isActive\", \"metadata\", \"createdAt\", \"updatedAt\") VALUES (DEFAULT, $1, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT) RETURNING \"id\", \"isActive\", \"createdAt\", \"updatedAt\"","routine":"ExecConstraints","schema":"public","severity":"ERROR","stack":"QueryFailedError: null value in column \"domain\" of relation \"companies\" violates not-null constraint\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:216:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async InsertQueryBuilder.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\query-builder\\InsertQueryBuilder.js:106:33)\n    at async SubjectExecutor.executeInsertOperations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\persistence\\SubjectExecutor.js:260:42)\n    at async SubjectExecutor.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\persistence\\SubjectExecutor.js:92:9)\n    at async EntityPersistExecutor.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\persistence\\EntityPersistExecutor.js:140:21)\n    at async AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:63:17)","table":"companies","timestamp":"2025-06-19T21:23:32.571Z"}
{"code":"23502","column":"domain","detail":"Failing row contains (8e893ca0-b4c1-4543-9667-5a91b8f32aec, Acme Corp, null, t, null, 2025-06-20 03:00:58.150241, 2025-06-20 03:00:58.150241).","driverError":{"code":"23502","column":"domain","detail":"Failing row contains (8e893ca0-b4c1-4543-9667-5a91b8f32aec, Acme Corp, null, t, null, 2025-06-20 03:00:58.150241, 2025-06-20 03:00:58.150241).","file":"execMain.c","length":316,"line":"1988","name":"error","routine":"ExecConstraints","schema":"public","severity":"ERROR","table":"companies"},"file":"execMain.c","length":316,"level":"error","line":"1988","message":"Registration error: null value in column \"domain\" of relation \"companies\" violates not-null constraint","parameters":["Acme Corp"],"query":"INSERT INTO \"companies\"(\"id\", \"name\", \"domain\", \"isActive\", \"metadata\", \"createdAt\", \"updatedAt\") VALUES (DEFAULT, $1, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT) RETURNING \"id\", \"isActive\", \"createdAt\", \"updatedAt\"","routine":"ExecConstraints","schema":"public","severity":"ERROR","stack":"QueryFailedError: null value in column \"domain\" of relation \"companies\" violates not-null constraint\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:216:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async InsertQueryBuilder.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\query-builder\\InsertQueryBuilder.js:106:33)\n    at async SubjectExecutor.executeInsertOperations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\persistence\\SubjectExecutor.js:260:42)\n    at async SubjectExecutor.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\persistence\\SubjectExecutor.js:92:9)\n    at async EntityPersistExecutor.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\persistence\\EntityPersistExecutor.js:140:21)\n    at async AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:63:17)","table":"companies","timestamp":"2025-06-19T21:30:58.181Z"}
{"0":{"context":{"child":"companyDomain","key":"companyDomain","label":"companyDomain","value":"acme.com"},"message":"\"companyDomain\" is not allowed","path":["companyDomain"],"type":"object.unknown"},"level":"error","message":"Validation error:","timestamp":"2025-06-19T21:34:47.503Z"}
{"code":"23502","column":"domain","detail":"Failing row contains (2a9759b1-85ad-4f21-8b19-3bae2dcccbea, Acme Corp, null, t, null, 2025-06-20 03:06:05.372302, 2025-06-20 03:06:05.372302).","driverError":{"code":"23502","column":"domain","detail":"Failing row contains (2a9759b1-85ad-4f21-8b19-3bae2dcccbea, Acme Corp, null, t, null, 2025-06-20 03:06:05.372302, 2025-06-20 03:06:05.372302).","file":"execMain.c","length":316,"line":"1988","name":"error","routine":"ExecConstraints","schema":"public","severity":"ERROR","table":"companies"},"file":"execMain.c","length":316,"level":"error","line":"1988","message":"Registration error: null value in column \"domain\" of relation \"companies\" violates not-null constraint","parameters":["Acme Corp"],"query":"INSERT INTO \"companies\"(\"id\", \"name\", \"domain\", \"isActive\", \"metadata\", \"createdAt\", \"updatedAt\") VALUES (DEFAULT, $1, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT) RETURNING \"id\", \"isActive\", \"createdAt\", \"updatedAt\"","routine":"ExecConstraints","schema":"public","severity":"ERROR","stack":"QueryFailedError: null value in column \"domain\" of relation \"companies\" violates not-null constraint\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:216:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async InsertQueryBuilder.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\query-builder\\InsertQueryBuilder.js:106:33)\n    at async SubjectExecutor.executeInsertOperations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\persistence\\SubjectExecutor.js:260:42)\n    at async SubjectExecutor.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\persistence\\SubjectExecutor.js:92:9)\n    at async EntityPersistExecutor.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\persistence\\EntityPersistExecutor.js:140:21)\n    at async AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:63:17)","table":"companies","timestamp":"2025-06-19T21:36:05.434Z"}
{"0":{"context":{"child":"companyDomain","key":"companyDomain","label":"companyDomain","value":"acme.com"},"message":"\"companyDomain\" is not allowed","path":["companyDomain"],"type":"object.unknown"},"level":"error","message":"Validation error:","timestamp":"2025-06-19T21:37:28.578Z"}
{"0":{"context":{"child":"companyDomain","key":"companyDomain","label":"companyDomain","value":"acme.com"},"message":"\"companyDomain\" is not allowed","path":["companyDomain"],"type":"object.unknown"},"level":"error","message":"Validation error:","timestamp":"2025-06-19T21:38:31.869Z"}
{"0":{"context":{"child":"companyDomain","key":"companyDomain","label":"companyDomain","value":"acme.com"},"message":"\"companyDomain\" is not allowed","path":["companyDomain"],"type":"object.unknown"},"level":"error","message":"Validation error:","timestamp":"2025-06-19T21:39:03.834Z"}
{"level":"error","message":"Error creating tenant database: QueryFailedError: syntax error at or near \"-\"","timestamp":"2025-06-19T21:41:46.010Z"}
{"code":"42601","driverError":{"code":"42601","file":"scan.l","length":90,"line":"1248","name":"error","position":"32","routine":"scanner_yyerror","severity":"ERROR"},"file":"scan.l","length":90,"level":"error","line":"1248","message":"Registration error: syntax error at or near \"-\"","position":"32","query":"CREATE DATABASE tenant_392bf3f9-1542-4bae-b259-1cd09ca72a41","routine":"scanner_yyerror","severity":"ERROR","stack":"QueryFailedError: syntax error at or near \"-\"\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\driver\\postgres\\PostgresQueryRunner.js:216:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async DataSource.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\data-source\\DataSource.js:353:20)\n    at async createTenantDatabase (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\config\\database.js:40:9)\n    at async AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\dist\\controllers\\auth.controller.js:66:17)","timestamp":"2025-06-19T21:41:46.018Z"}
{"level":"info","message":"Created database for tenant: tenant_d8f79afe2fe6485689fddcd75de25f04","timestamp":"2025-06-19T21:46:29.850Z"}
{"level":"info","message":"Created database for tenant: tenant_509a0bc0fe7a4f9f8d85487cb8c34f18","timestamp":"2025-06-26T16:07:31.398Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Error connecting to main database: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at async MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\migration\\MigrationExecutor.ts:336:17)\n    at async DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\data-source\\DataSource.ts:404:13)\n    at async DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\data-source\\DataSource.ts:273:17)\n    at async initializeMainDatabase (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\config\\database.ts:71:9)\n    at async startServer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\app.ts:72:9)","timestamp":"2025-06-26T16:24:01.272Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Error connecting to main database: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at async MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\migration\\MigrationExecutor.ts:336:17)\n    at async DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\data-source\\DataSource.ts:404:13)\n    at async DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\data-source\\DataSource.ts:273:17)\n    at async initializeMainDatabase (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\config\\database.ts:71:9)\n    at async startServer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\app.ts:72:9)","timestamp":"2025-06-26T16:24:12.442Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Error connecting to main database: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at async MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\migration\\MigrationExecutor.ts:336:17)\n    at async DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\data-source\\DataSource.ts:404:13)\n    at async DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\data-source\\DataSource.ts:273:17)\n    at async initializeMainDatabase (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\config\\database.ts:71:9)\n    at async startServer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\app.ts:72:9)","timestamp":"2025-06-26T16:24:28.714Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Error connecting to main database: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at async MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\migration\\MigrationExecutor.ts:336:17)\n    at async DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\data-source\\DataSource.ts:404:13)\n    at async DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\data-source\\DataSource.ts:273:17)\n    at async initializeMainDatabase (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\config\\database.ts:71:9)\n    at async startServer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\app.ts:72:9)","timestamp":"2025-06-26T16:25:03.738Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Error connecting to main database: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at async MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\migration\\MigrationExecutor.ts:336:17)\n    at async DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\data-source\\DataSource.ts:404:13)\n    at async DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\data-source\\DataSource.ts:273:17)\n    at async initializeMainDatabase (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\config\\database.ts:71:9)\n    at async startServer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\app.ts:72:9)","timestamp":"2025-06-26T16:25:22.942Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Error connecting to main database: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\typeorm\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at async MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\migration\\MigrationExecutor.ts:336:17)\n    at async DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\data-source\\DataSource.ts:404:13)\n    at async DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\src\\data-source\\DataSource.ts:273:17)\n    at async initializeMainDatabase (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\config\\database.ts:71:9)\n    at async startServer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\app.ts:72:9)","timestamp":"2025-06-26T16:25:55.432Z"}
{"level":"error","message":"Login error: No metadata for \"User\" was found.","stack":"EntityMetadataNotFoundError: No metadata for \"User\" was found.\n    at DataSource.getMetadata (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:451:30)\n    at Repository.get metadata [as metadata] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\repository\\Repository.ts:54:40)\n    at Repository.findOne (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\repository\\Repository.ts:626:42)\n    at AuthController.login (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\auth.controller.ts:97:10)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\middleware\\validation.middleware.ts:12:9\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express-rate-limit\\dist\\index.cjs:807:7","timestamp":"2025-06-26T16:28:46.780Z"}
{"level":"error","message":"Registration error: No metadata for \"Company\" was found.","stack":"EntityMetadataNotFoundError: No metadata for \"Company\" was found.\n    at DataSource.getMetadata (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:451:30)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\EntityPersistExecutor.ts:85:30\n    at Array.forEach (<anonymous>)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\EntityPersistExecutor.ts:77:30\n    at Array.map (<anonymous>)\n    at EntityPersistExecutor.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\EntityPersistExecutor.ts:73:34)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\auth.controller.ts:32:9)","timestamp":"2025-06-26T16:49:47.097Z"}
{"level":"error","message":"Registration error: No metadata for \"Company\" was found.","stack":"EntityMetadataNotFoundError: No metadata for \"Company\" was found.\n    at DataSource.getMetadata (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:451:30)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\EntityPersistExecutor.ts:85:30\n    at Array.forEach (<anonymous>)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\EntityPersistExecutor.ts:77:30\n    at Array.map (<anonymous>)\n    at EntityPersistExecutor.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\EntityPersistExecutor.ts:73:34)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\auth.controller.ts:32:9)","timestamp":"2025-06-26T16:49:48.482Z"}
{"level":"error","message":"Registration error: No metadata for \"Company\" was found.","stack":"EntityMetadataNotFoundError: No metadata for \"Company\" was found.\n    at DataSource.getMetadata (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:451:30)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\EntityPersistExecutor.ts:85:30\n    at Array.forEach (<anonymous>)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\EntityPersistExecutor.ts:77:30\n    at Array.map (<anonymous>)\n    at EntityPersistExecutor.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\EntityPersistExecutor.ts:73:34)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\auth.controller.ts:32:9)","timestamp":"2025-06-26T16:54:34.623Z"}
{"level":"error","message":"Registration error: No metadata for \"Company\" was found.","stack":"EntityMetadataNotFoundError: No metadata for \"Company\" was found.\n    at DataSource.getMetadata (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:451:30)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\EntityPersistExecutor.ts:85:30\n    at Array.forEach (<anonymous>)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\EntityPersistExecutor.ts:77:30\n    at Array.map (<anonymous>)\n    at EntityPersistExecutor.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\EntityPersistExecutor.ts:73:34)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\auth.controller.ts:32:9)","timestamp":"2025-06-26T17:08:51.736Z"}
{"level":"info","message":"Created database for tenant: tenant_56af2c6c3ee44a638781370b3d87b334","timestamp":"2025-06-26T17:15:38.154Z"}
{"code":"23505","constraint":"UQ_89a223b4d883067d909eedd3558","detail":"Key (domain)=(acme.com) already exists.","driverError":{"code":"23505","constraint":"UQ_89a223b4d883067d909eedd3558","detail":"Key (domain)=(acme.com) already exists.","file":"nbtinsert.c","length":235,"line":"673","name":"error","routine":"_bt_check_unique","schema":"public","severity":"ERROR","table":"companies"},"file":"nbtinsert.c","length":235,"level":"error","line":"673","message":"Registration error: duplicate key value violates unique constraint \"UQ_89a223b4d883067d909eedd3558\"","parameters":["Acme Inc","acme.com"],"query":"INSERT INTO \"companies\"(\"id\", \"name\", \"domain\", \"isActive\", \"metadata\", \"createdAt\", \"updatedAt\") VALUES (DEFAULT, $1, $2, DEFAULT, DEFAULT, DEFAULT, DEFAULT) RETURNING \"id\", \"isActive\", \"createdAt\", \"updatedAt\"","routine":"_bt_check_unique","schema":"public","severity":"ERROR","stack":"QueryFailedError: duplicate key value violates unique constraint \"UQ_89a223b4d883067d909eedd3558\"\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at InsertQueryBuilder.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\query-builder\\InsertQueryBuilder.ts:163:33)\n    at SubjectExecutor.executeInsertOperations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\SubjectExecutor.ts:435:42)\n    at SubjectExecutor.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\SubjectExecutor.ts:137:9)\n    at EntityPersistExecutor.execute (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\persistence\\EntityPersistExecutor.ts:182:21)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\auth.controller.ts:32:9)","table":"companies","timestamp":"2025-06-26T18:49:38.062Z"}
{"level":"info","message":"Created database for tenant: tenant_6916f045ff604738a7beaf8015e51d9f","timestamp":"2025-06-26T18:50:23.131Z"}
{"level":"error","message":"Registration error: Role already exists","name":"AuthError","stack":"AuthError: Role already exists\n    at AuthService.createRole (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\services\\auth.service.ts:198:13)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\auth.controller.ts:45:27)","statusCode":400,"timestamp":"2025-06-26T18:50:23.545Z"}
{"level":"info","message":"Created database for tenant: tenant_18d3959e6de947a09895cb505285772a","timestamp":"2025-06-26T18:51:21.740Z"}
{"level":"error","message":"Registration error: Role already exists","name":"AuthError","stack":"AuthError: Role already exists\n    at AuthService.createRole (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\services\\auth.service.ts:198:13)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at AuthController.register (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\auth.controller.ts:45:27)","statusCode":400,"timestamp":"2025-06-26T18:51:22.085Z"}
{"level":"info","message":"Created database for tenant: tenant_f3e1f6eec36f42029aa317619538f310","timestamp":"2025-06-26T19:33:17.673Z"}
{"level":"info","message":"Created database for tenant: tenant_f830d11973a24393b5563c5c947a1d22","timestamp":"2025-06-26T20:19:39.116Z"}
{"level":"info","message":"Created database for tenant: tenant_a782d35d01264fa39589f960a3ff94a7","timestamp":"2025-06-26T20:30:06.876Z"}
{"level":"error","message":"Error creating tenant database: TypeORMError: Entity metadata for users_client#company was not found. Check if you specified a correct entity object and if it's connected in the connection options.","timestamp":"2025-06-26T20:30:06.927Z"}
{"level":"error","message":"Registration error: Entity metadata for users_client#company was not found. Check if you specified a correct entity object and if it's connected in the connection options.","stack":"TypeORMError: Entity metadata for users_client#company was not found. Check if you specified a correct entity object and if it's connected in the connection options.\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\metadata-builder\\EntityMetadataBuilder.ts:1128:23\n    at Array.forEach (<anonymous>)\n    at EntityMetadataBuilder.computeInverseProperties (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\metadata-builder\\EntityMetadataBuilder.ts:1118:34)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\metadata-builder\\EntityMetadataBuilder.ts:160:18\n    at Array.forEach (<anonymous>)\n    at EntityMetadataBuilder.build (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\metadata-builder\\EntityMetadataBuilder.ts:159:25)\n    at ConnectionMetadataBuilder.buildEntityMetadatas (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\connection\\ConnectionMetadataBuilder.ts:106:11)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at DataSource.buildMetadatas (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:733:13)\n    at DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:264:13)","timestamp":"2025-06-26T20:30:06.953Z"}
{"level":"info","message":"Created database for tenant: tenant_b4f42dc4c3344ebfa57b64dbad5c8996","timestamp":"2025-06-26T20:30:19.539Z"}
{"level":"error","message":"Error creating tenant database: TypeORMError: Entity metadata for users_client#company was not found. Check if you specified a correct entity object and if it's connected in the connection options.","timestamp":"2025-06-26T20:30:19.696Z"}
{"level":"error","message":"Registration error: Entity metadata for users_client#company was not found. Check if you specified a correct entity object and if it's connected in the connection options.","stack":"TypeORMError: Entity metadata for users_client#company was not found. Check if you specified a correct entity object and if it's connected in the connection options.\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\metadata-builder\\EntityMetadataBuilder.ts:1128:23\n    at Array.forEach (<anonymous>)\n    at EntityMetadataBuilder.computeInverseProperties (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\metadata-builder\\EntityMetadataBuilder.ts:1118:34)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\metadata-builder\\EntityMetadataBuilder.ts:160:18\n    at Array.forEach (<anonymous>)\n    at EntityMetadataBuilder.build (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\metadata-builder\\EntityMetadataBuilder.ts:159:25)\n    at ConnectionMetadataBuilder.buildEntityMetadatas (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\connection\\ConnectionMetadataBuilder.ts:106:11)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at DataSource.buildMetadatas (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:733:13)\n    at DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:264:13)","timestamp":"2025-06-26T20:30:19.698Z"}
{"level":"error","message":"Customer registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresDriver.ts:1157:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:111:18)\n    at CustomerController.registerCustomer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\customer.controller.ts:16:25)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\index.js:284:15\n    at Function.process_params (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\index.js:280:10)","timestamp":"2025-06-27T05:56:54.707Z"}
{"level":"error","message":"Customer registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresDriver.ts:1157:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:111:18)\n    at CustomerController.registerCustomer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\customer.controller.ts:16:25)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\index.js:284:15\n    at Function.process_params (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\index.js:280:10)","timestamp":"2025-06-27T05:57:22.115Z"}
{"level":"error","message":"Customer registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresDriver.ts:1157:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:111:18)\n    at CustomerController.registerCustomer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\customer.controller.ts:16:25)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\index.js:284:15\n    at Function.process_params (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\index.js:280:10)","timestamp":"2025-06-27T05:58:30.915Z"}
{"level":"error","message":"Customer registration error: Driver not Connected","stack":"TypeORMError: Driver not Connected\n    at PostgresDriver.obtainMasterConnection (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresDriver.ts:1157:19)\n    at PostgresQueryRunner.connect (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:111:18)\n    at CustomerController.registerCustomer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\customer.controller.ts:16:25)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\index.js:284:15\n    at Function.process_params (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\node_modules\\express\\lib\\router\\index.js:280:10)","timestamp":"2025-06-27T06:01:29.320Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Customer registration error: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migration\\MigrationExecutor.ts:336:17)\n    at DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:404:13)\n    at DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:273:17)\n    at CustomerController.registerCustomer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\customer.controller.ts:14:9)","timestamp":"2025-06-27T06:04:59.165Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Customer registration error: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migration\\MigrationExecutor.ts:336:17)\n    at DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:404:13)\n    at DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:273:17)\n    at CustomerController.registerCustomer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\customer.controller.ts:14:9)","timestamp":"2025-06-27T06:06:26.203Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Customer registration error: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migration\\MigrationExecutor.ts:336:17)\n    at DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:404:13)\n    at DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:273:17)\n    at CustomerController.registerCustomer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\customer.controller.ts:14:9)","timestamp":"2025-06-27T06:07:59.446Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Customer registration error: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migration\\MigrationExecutor.ts:336:17)\n    at DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:404:13)\n    at DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:273:17)\n    at CustomerController.registerCustomer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\customer.controller.ts:14:9)","timestamp":"2025-06-27T06:13:36.856Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Customer registration error: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migration\\MigrationExecutor.ts:336:17)\n    at DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:404:13)\n    at DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:273:17)\n    at CustomerController.registerCustomer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\customer.controller.ts:14:9)","timestamp":"2025-06-27T06:22:30.602Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Customer registration error: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migration\\MigrationExecutor.ts:336:17)\n    at DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:404:13)\n    at DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:273:17)\n    at CustomerController.registerCustomer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\customer.controller.ts:14:9)","timestamp":"2025-06-27T06:29:08.160Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Customer registration error: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migration\\MigrationExecutor.ts:336:17)\n    at DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:404:13)\n    at DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:273:17)\n    at CustomerController.registerCustomer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\customer.controller.ts:14:9)","timestamp":"2025-06-27T11:42:11.249Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Customer registration error: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migration\\MigrationExecutor.ts:336:17)\n    at DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:404:13)\n    at DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:273:17)\n    at CustomerController.registerCustomer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\customer.controller.ts:14:9)","timestamp":"2025-06-27T11:49:41.767Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Customer registration error: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migration\\MigrationExecutor.ts:336:17)\n    at DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:404:13)\n    at DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:273:17)\n    at CustomerController.registerCustomer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\customer.controller.ts:14:9)","timestamp":"2025-06-27T12:30:40.372Z"}
{"code":"42P07","driverError":{"code":"42P07","file":"heap.c","length":101,"line":"1162","name":"error","routine":"heap_create_with_catalog","severity":"ERROR"},"file":"heap.c","length":101,"level":"error","line":"1162","message":"Customer registration error: relation \"modules\" already exists","query":"CREATE TABLE \"modules\" (\"id\" uuid NOT NULL DEFAULT uuid_generate_v4(), \"name\" character varying NOT NULL, \"version\" character varying NOT NULL, \"description\" character varying NOT NULL, \"isActive\" boolean NOT NULL DEFAULT true, \"config\" jsonb NOT NULL, \"createdAt\" TIMESTAMP NOT NULL DEFAULT now(), \"updatedAt\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_8cd1abde4b70e59644c98668c06\" UNIQUE (\"name\"), CONSTRAINT \"PK_7dbefd488bd96c5bf31f0ce0c95\" PRIMARY KEY (\"id\"))","routine":"heap_create_with_catalog","severity":"ERROR","stack":"QueryFailedError: relation \"modules\" already exists\n    at PostgresQueryRunner.query (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\driver\\postgres\\PostgresQueryRunner.ts:325:19)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at InitialMigration1750367465052.up (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migrations\\1750367465052-InitialMigration.ts:7:9)\n    at MigrationExecutor.executePendingMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\migration\\MigrationExecutor.ts:336:17)\n    at DataSource.runMigrations (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:404:13)\n    at DataSource.initialize (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\data-source\\DataSource.ts:273:17)\n    at CustomerController.registerCustomer (C:\\Users\\ekana\\OneDrive\\Desktop\\nodejs\\AdminEngine\\src\\controllers\\customer.controller.ts:14:9)","timestamp":"2025-06-27T12:35:37.686Z"}
